ui/app/src/services/workflow.ts

```diff
@@ -90,17 +90,14 @@ export function requestPublish(siteId: string, data: PublishingParams): Observab
   }).pipe(map(() => true));
 }
 
-export function approve(siteId: string, data: PublishingParams): Observable<boolean> {
-  return postJSON('/studio/api/2/workflow/approve', {
-    siteId,
+export function approve(siteId: string, packageId: number, data: ApproveParams): Observable<boolean> {
```

Comment:
Change to return Observable<WhatEverBackReturns> and remove the map for all new. Observable<void> if they don't actually return anything.

Reply:
approve and reject don't return anything, leaving the Observable<boolean>